Python/Product/PythonTools/PythonTools/DiagnosticsProvider.cs

```diff
@@ -176,6 +177,65 @@ internal sealed class DiagnosticsProvider {
+            writer.WriteLine("    EnterCommitsIntellisense: {0}", pyService.AdvancedOptions.EnterCommitsIntellisense);
+            writer.WriteLine("    EnterOutliningModeOnOpen: {0}", pyService.AdvancedOptions.EnterOutliningModeOnOpen);
+            writer.WriteLine("    FilterCompletions: {0}", pyService.AdvancedOptions.FilterCompletions);
+            writer.WriteLine("    HideAdvancedMembers: {0}", pyService.AdvancedOptions.HideAdvancedMembers);
+            writer.WriteLine("    IntersectMembers: {0}", pyService.AdvancedOptions.IntersectMembers);
+            writer.WriteLine("    PasteRemovesReplPrompts: {0}", pyService.AdvancedOptions.PasteRemovesReplPrompts);
+            writer.WriteLine();
+
+            writer.WriteLine("Debugger Options:");
+            writer.WriteLine("    BreakOnSystemExitZero: {0}", pyService.DebuggerOptions.BreakOnSystemExitZero);
+            writer.WriteLine("    DebugStdLib: {0}", pyService.DebuggerOptions.DebugStdLib);
+            writer.WriteLine("    PromptBeforeRunningWithBuildError: {0}", pyService.DebuggerOptions.PromptBeforeRunningWithBuildError);
+            writer.WriteLine("    ShowFunctionReturnValue: {0}", pyService.DebuggerOptions.ShowFunctionReturnValue);
+            writer.WriteLine("    TeeStandardOutput: {0}", pyService.DebuggerOptions.TeeStandardOutput);
+            writer.WriteLine("    UseLegacyDebugger: {0}", pyService.DebuggerOptions.UseLegacyDebugger);
+            writer.WriteLine("    WaitOnAbnormalExit: {0}", pyService.DebuggerOptions.WaitOnAbnormalExit);
+            writer.WriteLine("    WaitOnNormalExit: {0}", pyService.DebuggerOptions.WaitOnNormalExit);
+            writer.WriteLine();
+
+            writer.WriteLine("Conda Options:");
+            writer.WriteLine("    CustomCondaExecutablePath: {0}", pyService.CondaOptions.CustomCondaExecutablePath);
+            writer.WriteLine();
+
+            writer.WriteLine("Interactive Options:");
+            writer.WriteLine("    CompletionMode: {0}", pyService.InteractiveOptions.CompletionMode);
+            writer.WriteLine("    LiveCompletionsOnly: {0}", pyService.InteractiveOptions.LiveCompletionsOnly);
+            writer.WriteLine("    Scripts: {0}", pyService.InteractiveOptions.Scripts);
+            writer.WriteLine("    UseSmartHistory: {0}", pyService.InteractiveOptions.UseSmartHistory);
+            writer.WriteLine();
+
+            writer.WriteLine("Debug Interactive Options:");
+            writer.WriteLine("    CompletionMode: {0}", pyService.DebugInteractiveOptions.CompletionMode);
+            writer.WriteLine("    LiveCompletionsOnly: {0}", pyService.DebugInteractiveOptions.LiveCompletionsOnly);
+            writer.WriteLine("    Scripts: {0}", pyService.DebugInteractiveOptions.Scripts);
+            writer.WriteLine("    UseSmartHistory: {0}", pyService.DebugInteractiveOptions.UseSmartHistory);
+            writer.WriteLine();
+
```

Comment:
I wonder if it worth making these options available as a list?

Reply:
I don't foresee other scenarios needing to do this now, but if that comes up again we could introduce a list then.